feat: update/add pageSeparator params to LlamaParseReader (#1051)
run-llama/LlamaIndexTS: apps/docs/docs/modules/data_loaders/llama_parse/index.mdx

- `doNotCache?` Optional. Set to true to not cache the document.
- `fastMode?` Optional. Set to true to use the fast mode. This mode will skip OCR of images, and table/heading reconstruction. Note: Non-compatible with `gpt4oMode`.
- `doNotUnrollColumns?` Optional. Set to true to keep the text according to document layout. Reduce reconstruction accuracy, and LLMs/embeddings performances in most cases.

> - `pageSeparator?` Optional. A templated page separator to use to split the text. If it contain `{page_number}`, it will be replaced by the next page number. If not set the default separator `\\n---\\n` will be used.

Reviewer: Why does the `pageSeparator` contain the `{page_number}`?

Author: Sorry, I just copied the python description. 
I assume it's currently only relevant for JSON mode where a page_number is returned.
Improved the descriptions.

> - `pagePrefix?` Optional. A templated prefix to add to the beginning of each page. If it contain `{page_number}`, it will be replaced by the page number.
> - `pageSuffix?` Optional. A templated suffix to add to the end of each page. If it contain `{page_number}`, it will be replaced by the page number.

Reviewer: ```suggestion
- `pagePrefix?` Optional. A templated prefix to add to the beginning of each page. The variable `{page_number}` will be replaced by the page number.
- `pageSuffix?` Optional. A templated suffix to add to the end of each page. The variable `{page_number}` will be replaced by the page number.
```
